src/components/ha-form/ha-form.ts

```diff
@@ -85,14 +92,16 @@ export type HaFormData =
   | HaFormBooleanData
   | HaFormSelectData
   | HaFormMultiSelectData
-  | HaFormTimeData;
+  | HaFormTimeData
+  | HaFormDictionaryData;
 
 export type HaFormStringData = string;
 export type HaFormIntegerData = number;
 export type HaFormFloatData = number;
 export type HaFormBooleanData = boolean;
 export type HaFormSelectData = string;
 export type HaFormMultiSelectData = string[];
+export type HaFormDictionaryData = { [key: string]: HaFormData };
```

Comment:
This is the same as a default data container?

Reply:
the dictionary is basically nested as an internal form, so this makes sense. will change

Reply:
done